go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/opendatahub-io/odh-model-controller
 
-go 1.23.6
+go 1.24.4
```

Comment:
did you double check that 
https://github.com/red-hat-data-services/odh-model-controller/blob/f723cace036985635055c6a4210775ffab871a85/Dockerfile.konflux#L6C6-L6C53
is at least 1.24.4?

Reply:
is it weird that I'm not finding the `sha256:d1c1f2af6122b0ea3456844210d6d9c4d96e78c128bc02fabeadf51239202221` in https://catalog.redhat.com/en/software/containers/ubi9/go-toolset/61e5c00b4ec9945c18787690/history to reference the version?